### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by Tuan1O2**
- add libs: classnames + prop-types + react-flags-select
- config files: eslintrc.cjs + tailwind.config.js + postcss.config.js
- create components: Button + Header + LinkBtn
- create Welcome page (make default instead of Login) in auth folder + responsive Header + Welcome
- add imgs + create constants imgUrl

### CODE REVIEW

**Comment by MaiNhatHoangY2001** [comment]
Please change commit to conventional commit
image

File: src/components/common/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+interface Props {
+  className: string;
+  children?: React.ReactNode;
+  onClick: () => void;
+}
+
+const Button: React.FC<Props> = ({className, children, onClick}) => {
+  return (
+    <button
```
Comment (MaiNhatHoangY2001) [inline src/components/common/Button.tsx]: Why are you not using material-ui ?

File: tailwind.config.js [hunk 2]
```diff
@@ -14,6 +14,9 @@ export default {
       "accent-color4": "#174444",
       "static-color": "#F7F7F7",
     },
+    fontFamily: {
+      roboto: ["Roboto"],
```
Comment (MaiNhatHoangY2001) [inline tailwind.config.js]: Use the font family from the variables.css, dont used by Tailwind; it is used for maintenance if we want to alter the typeface.

File: src/layouts/PublicLayout.tsx
Comment (mailong395) [inline src/layouts/PublicLayout.tsx:1]: page review is a private page, pls fix it.

File: src/pages/auth/Welcome.tsx
Comment (mailong395) [inline src/pages/auth/Welcome.tsx:1]: the header compoennt should be placed in the layout component.

**Review by MaiNhatHoangY2001 — CHANGES_REQUESTED** [review]

File: src/routers/AppRoutes.tsx
Comment (mailong395) [inline src/routers/AppRoutes.tsx:1]: page review is a private page, pls fix it.

File: src/routers/PrivateRoute.tsx
Comment (mailong395) [inline src/routers/PrivateRoute.tsx:1]: page review is a private page, pls fix it.

**Review by mailong395 — COMMENTED** [review]
Please fix the issues I commented

File: src/pages/auth/Welcome.tsx
Comment (Tuan1O2) [reply src/pages/auth/Welcome.tsx:1]: I've already put it right, pls check again
image

**Comment by MaiNhatHoangY2001** [comment]
image
Please combine all commit to one and commit message is "feat: Add welcome page and header layout"

File: src/styles/variables.scss [hunk 3]
```diff
@@ -8,4 +8,5 @@
   --accent-color3: #c7f9cc;
   --accent-color4: #174444;
   --static-color: #f7f7f7;
+  --robotoFont: "Roboto";
```
Comment (MaiNhatHoangY2001) [inline src/styles/variables.scss]:
```suggestion
  --font-family: "Roboto";
```
Please change to this

**Review by MaiNhatHoangY2001 — APPROVED** [review]

**Pull request closed without merging**